typings/index.d.ts

```diff
@@ -3061,6 +3061,13 @@ declare module 'discord.js' {
     large_threshold?: number;
     compress?: boolean;
     intents?: BitFieldResolvable<IntentsString> | number;
+    properties?: WebSocketProperties;
```

Comment:
Why not type this as a `Record<string, unknown>` instead? Then you won't need extra PRs if you want to add anything else

Reply:
I'd argue it didn't change much in the last 2 years, and sacrificing accurate typings for not making more PRs later on is a trade-off.